internal/commands/commands.go

```diff
@@ -12,13 +12,15 @@ const (
 	LostSector     string = "lostsector"
 	LostSectorList string = "lostsectorlist"
 	Nightfall      string = "nightfall"
+	Dungeon		   string = "dungeon"	
```

Comment:
very minor; make spacing consistent

```
LostSectorList string = "lostsectorlist"
Nightfall      string = "nightfall"
Dungeon        string = "dungeon"	
```

Reply:
Odd, looks fine on mine. Must be a spacing vs tabbing issue. Yep, that's what it was